dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -56,19 +159,92 @@ public SauceOptions()
         public string TimeZone { get; set; }
         public string TunnelIdentifier { get; set; }
         public bool VideoUploadOnPass { get; set; }
-
-        public void WithEdge()
+        public DriverOptions SeleniumOptions { get; set; }
+        public Browser BrowserName { get; set; } = Browser.Chrome;
+        //TODO not fond of this name
+        public static List<string> W3CAllowedOptionsList = new List<string>(new string[]
+            {
+                "BrowserName",
+                "BrowserVersion",
+                "PlatformName",
+                "PageLoadStrategy",
+                "AcceptInsecureCerts",
+                "Proxy",
+                "SetWindowRect",
+                "Timeouts",
+                "StrictFileInteractability",
+                "UnhandledPromptBehavior"
+            }
+        );
+        private static readonly List<string> _sauceAllowedOptions = new List<string>(new string[]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field '_sauceAllowedOptions'.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4945851)